### PR TITLE
Updated: Cryptpad config

### PR DESCRIPTION
**Pull request by GoliathLabs**
I've tried setting the proper headers as requested.

### CODE REVIEW

**Comment by goligo** [comment]
Did you test, whether cryptpad still works with these changes?

**Comment by GoliathLabs** [comment]
Not yet, but I don't see a reason why it shouldn't (as we're only adding a few headers which we're recommended by the upstream docs) @goligo

**Comment by goligo** [comment]
Just adding 50 lines of code setting CORS and CSP headers without understanding what they actually do - what could possibly go wrong?

**Comment by goligo** [comment]
CORS and CSP are quite complex and they have an impact on security. So at least you should provide information on where you found the inserted code and you need to test, whether Cryptpad still works with these headers.

**Comment by goligo** [comment]
Why would you just close it? I guess there was a good reason to change the config for cryptpad?

**Comment by goligo** [comment]
https://github.com/xwiki-labs/cryptpad/blob/main/docs/example.nginx.conf

**Comment by awlx** [comment]
Thank you @goligo for testing this. For me it looks good to go. The headers are once for `embedder` and once for `resource` which is probably set by the application?

**Comment by goligo** [comment]
I think the change is unnecessary - it looks like Cryptpad already takes care of setting the appropriate headers, there is no need to include them in our nginx configuration?

**Comment by awlx** [comment]
I think this is all related to this conversation:
https://chat.ffmuc.net/freifunk/pl/f4hktu9etfgcjg9t4fqakkp79a

**Comment by awlx** [comment]
And those headers fail:
https://cryptpad.ffmuc.net/checkup/

**Comment by awlx** [comment]
TL;DR it's necessary if we want to be listed on the instance page [here](https://cryptpad.org/instances/).

**Comment by GoliathLabs** [comment]
I've updated the nginx config. Feel free to leave some feedback

**Comment by GoliathLabs** [comment]
Most headers have been set correctly by cryptpad already. We only need to set one header to get the checklist done. But I will do this in another PR
